scripts/dts/edtlib.py

```diff
@@ -8,18 +8,20 @@
 Library for working with devicetrees at a higher level compared to dtlib. Like
 dtlib, this library presents a tree of devicetree nodes, but the nodes are
 augmented with information from bindings and include some interpretation of
-properties.
+properties. Some of this interpretation is based on conventions established
+by the Linux kernel, so the Documentation/devicetree/bindings in the Linux
+source code is sometimes good reference material.
 
-Bindings are files that describe devicetree nodes. Devicetree nodes are usually
-mapped to bindings via their 'compatible = "..."' property, but a binding can
-also come from a 'child-binding:' key in the binding for the parent devicetree
-node.
+In Zephyr, bindings are YAML files that describe devicetree nodes. Devicetree
+nodes are usually mapped to bindings via their 'compatible = "..."' property,
+but a binding can also come from a 'child-binding:' key in the binding for the
+parent devicetree node. See binding-template.yaml for details.
```

Comment:
"See binding-template.yaml " makes binding-template.yaml kind of part of edtlib.py. As binding-template.yaml is not stored besides edtlib.py the full path would be more informative "[zephyr]/dts/bindings-template.yaml". For further improvement it could become part of the edtlib.py "library".

Reply:
> "See binding-template.yaml " makes binding-template.yaml kind of part of edtlib.py.

Yeah, I'll just drop this for now. I want to deal with that file in a separate PR if that's OK with you.